app/assets/javascripts/modules/calendars/guider-slot-picker.es6

```diff
@@ -168,13 +168,15 @@
         eventsToAdd = [];
 
       for (let currentDate = moment(calendarStartDate); currentDate < calendarEndDate; currentDate.add(1, 'days')) {
-        for (let eventIndex in events) {
-          let currentEvent = events[eventIndex];
+        if (currentDate.day() != 0 && currentDate.day() != 6) {
```

Comment:
I feel we should call this logic out somewhere in one of the feature tests since we have no idea this works unless I boot the app up and step through manually.